2019-03-14 20:31:18,444-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:18,516-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'data': None, 'code': '20110', 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:31:18,524-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'code': '10001', 'status': 1, 'msg': '注册成功'} != {'data': None, 'code': '20110', 'status': 0, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:31:18,555-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:18,597-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:18,694-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20108', 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:31:18,716-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:18,724-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:18,812-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20108', 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:31:18,833-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:18,843-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:18,906-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'data': None, 'code': '20109', 'status': 0, 'msg': '手机号码格式不正确'}
2019-03-14 20:31:18,926-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:18,936-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:19,006-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'data': None, 'code': '20110', 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:31:19,030-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:19,039-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:19,108-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'data': None, 'code': '10001', 'status': 1, 'msg': '登录成功'}
2019-03-14 20:31:19,130-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:19,142-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:19,219-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'data': None, 'code': '20111', 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:31:19,238-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:19,253-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:19,329-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'data': None, 'code': '20111', 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:31:19,352-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:19,363-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:19,433-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'code': '20103', 'status': 0, 'msg': '手机号不能为空'}
2019-03-14 20:31:19,454-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:19,463-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:19,528-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20103', 'status': 0, 'msg': '密码不能为空'}
2019-03-14 20:31:19,549-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:42,302-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:42,374-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'pwd': '123456', 'regname': '自闭患者'}，结果是{'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
2019-03-14 20:31:42,381-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'data': None, 'code': '10001', 'status': 1} != {'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:31:42,452-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:42,484-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:42,578-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '12345', 'regname': '哎呀'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'code': '20108', 'status': 0}
2019-03-14 20:31:42,635-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:42,645-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:42,709-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '1234567891234567891', 'regname': '妈呀'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'code': '20108', 'status': 0}
2019-03-14 20:31:42,761-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:42,774-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:42,839-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'pwd': '123456', 'regname': '小皮球'}，结果是{'msg': '手机号码格式不正确', 'data': None, 'code': '20109', 'status': 0}
2019-03-14 20:31:42,893-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:42,904-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:42,975-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'pwd': '123456', 'regname': '小晚晴'}，结果是{'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
2019-03-14 20:31:43,027-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:43,036-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:43,118-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'msg': '登录成功', 'data': None, 'code': '10001', 'status': 1}
2019-03-14 20:31:43,172-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:43,182-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:43,250-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'data': None, 'code': '20111', 'status': 0}
2019-03-14 20:31:43,301-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:43,309-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:43,384-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'data': None, 'code': '20111', 'status': 0}
2019-03-14 20:31:43,441-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:43,449-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:43,521-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'data': None, 'code': '20103', 'status': 0}
2019-03-14 20:31:43,574-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:31:43,583-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:31:43,649-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'data': None, 'code': '20103', 'status': 0}
2019-03-14 20:31:43,702-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:47,292-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:47,365-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:32:47,373-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'data': None, 'status': 1, 'msg': '注册成功'} != {'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:32:47,432-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:47,444-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:47,508-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'code': '20108', 'data': None, 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:32:47,562-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:47,570-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:47,638-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'code': '20108', 'data': None, 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:32:47,695-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:47,704-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:47,773-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'code': '20109', 'data': None, 'status': 0, 'msg': '手机号码格式不正确'}
2019-03-14 20:32:47,845-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:47,854-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:47,919-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:32:47,966-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:47,974-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:48,086-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'data': None, 'status': 1, 'msg': '登录成功'}
2019-03-14 20:32:48,143-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:48,151-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:48,263-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'data': None, 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:32:48,316-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:48,325-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:48,396-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'data': None, 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:32:48,445-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:48,453-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:48,521-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'data': None, 'status': 0, 'msg': '手机号不能为空'}
2019-03-14 20:32:48,578-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:32:48,587-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:32:48,655-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'data': None, 'status': 0, 'msg': '密码不能为空'}
2019-03-14 20:32:48,715-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:44,296-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:44,368-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:34:44,376-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'data': None, 'code': '10001', 'msg': '注册成功'} != {'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:34:44,444-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:44,460-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:44,533-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:34:44,584-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:44,593-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:44,655-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:34:44,703-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:44,714-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:44,778-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 20:34:44,830-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:44,839-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:44,908-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:34:44,961-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:44,970-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:45,044-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'status': 1, 'data': None, 'code': '10001', 'msg': '登录成功'}
2019-03-14 20:34:45,095-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:45,103-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:45,181-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:34:45,236-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:45,245-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:45,321-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:34:45,369-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:45,378-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:45,447-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 20:34:45,498-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:34:45,506-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:34:45,576-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 20:34:45,629-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:03,967-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,071-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:36:04,085-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'data': None, 'status': 1, 'msg': '注册成功'} != {'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:36:04,142-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,154-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,247-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'code': '20108', 'data': None, 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:36:04,298-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,307-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,371-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'code': '20108', 'data': None, 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:36:04,428-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,438-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,505-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'code': '20109', 'data': None, 'status': 0, 'msg': '手机号码格式不正确'}
2019-03-14 20:36:04,561-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,569-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,668-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:36:04,718-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,726-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,799-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'code': '10001', 'data': None, 'status': 1, 'msg': '登录成功'}
2019-03-14 20:36:04,849-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,858-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:04,928-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'code': '20111', 'data': None, 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:36:04,985-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:04,996-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:05,067-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'code': '20111', 'data': None, 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:36:05,121-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:05,129-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:05,199-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'data': None, 'status': 0, 'msg': '手机号不能为空'}
2019-03-14 20:36:05,247-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:05,259-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:05,326-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'data': None, 'status': 0, 'msg': '密码不能为空'}
2019-03-14 20:36:05,381-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:47,496-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:47,573-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'code': '20110', 'status': 0, 'msg': '手机号码已被注册', 'data': None}
2019-03-14 20:36:47,586-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'status': 1, 'msg': '注册成功', 'data': None} != {'code': '20110', 'status': 0, 'msg': '手机号码已被注册', 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:36:47,653-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:47,664-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:47,729-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'code': '20108', 'status': 0, 'msg': '密码长度必须为6~18', 'data': None}
2019-03-14 20:36:47,779-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:47,788-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:47,853-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'code': '20108', 'status': 0, 'msg': '密码长度必须为6~18', 'data': None}
2019-03-14 20:36:47,901-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:47,909-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:47,976-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'code': '20109', 'status': 0, 'msg': '手机号码格式不正确', 'data': None}
2019-03-14 20:36:48,033-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:48,041-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:48,111-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'code': '20110', 'status': 0, 'msg': '手机号码已被注册', 'data': None}
2019-03-14 20:36:48,167-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:48,176-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:48,246-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'code': '10001', 'status': 1, 'msg': '登录成功', 'data': None}
2019-03-14 20:36:48,299-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:48,308-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:48,380-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'code': '20111', 'status': 0, 'msg': '用户名或密码错误', 'data': None}
2019-03-14 20:36:48,435-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:48,444-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:48,515-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'code': '20111', 'status': 0, 'msg': '用户名或密码错误', 'data': None}
2019-03-14 20:36:48,567-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:48,576-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:48,640-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'status': 0, 'msg': '手机号不能为空', 'data': None}
2019-03-14 20:36:48,695-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:36:48,703-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:36:48,774-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'status': 0, 'msg': '密码不能为空', 'data': None}
2019-03-14 20:36:48,828-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:28,953-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,026-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:37:29,033-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'status': 1, 'code': '10001', 'msg': '注册成功'} != {'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:37:29,092-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,104-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,203-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:37:29,253-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,264-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,334-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:37:29,386-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,395-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,468-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 20:37:29,518-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,527-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,597-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:37:29,648-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,656-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,760-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'status': 1, 'code': '10001', 'msg': '登录成功'}
2019-03-14 20:37:29,820-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,830-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:29,905-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:37:29,966-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:29,974-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:30,047-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:37:30,120-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:30,128-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:30,196-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 20:37:30,261-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:37:30,269-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:37:30,342-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 20:37:30,392-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:15,644-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:15,717-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'pwd': '123456', 'mobilephone': '13127750530'}，结果是{'status': 0, 'msg': '手机号码已被注册', 'data': None, 'code': '20110'}
2019-03-14 20:38:15,724-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'msg': '注册成功', 'data': None, 'code': '10001'} != {'status': 0, 'msg': '手机号码已被注册', 'data': None, 'code': '20110'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:38:15,790-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:15,807-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,112-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'pwd': '12345', 'mobilephone': '13412345678'}，结果是{'status': 0, 'msg': '密码长度必须为6~18', 'data': None, 'code': '20108'}
2019-03-14 20:38:16,165-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,174-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,242-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'pwd': '1234567891234567891', 'mobilephone': '13412345678'}，结果是{'status': 0, 'msg': '密码长度必须为6~18', 'data': None, 'code': '20108'}
2019-03-14 20:38:16,295-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,306-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,378-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'pwd': '123456', 'mobilephone': 'abcd'}，结果是{'status': 0, 'msg': '手机号码格式不正确', 'data': None, 'code': '20109'}
2019-03-14 20:38:16,426-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,435-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,508-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'pwd': '123456', 'mobilephone': '13412345679'}，结果是{'status': 0, 'msg': '手机号码已被注册', 'data': None, 'code': '20110'}
2019-03-14 20:38:16,576-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,584-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,653-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'msg': '登录成功', 'data': None, 'code': '10001'}
2019-03-14 20:38:16,702-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,710-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,786-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'msg': '用户名或密码错误', 'data': None, 'code': '20111'}
2019-03-14 20:38:16,844-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,852-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:16,922-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'msg': '用户名或密码错误', 'data': None, 'code': '20111'}
2019-03-14 20:38:16,972-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:16,980-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:17,077-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'msg': '手机号不能为空', 'data': None, 'code': '20103'}
2019-03-14 20:38:17,129-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:17,137-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:17,203-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'msg': '密码不能为空', 'data': None, 'code': '20103'}
2019-03-14 20:38:17,253-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:55,796-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:55,868-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'pwd': '123456', 'mobilephone': '13127750530'}，结果是{'data': None, 'code': '20110', 'msg': '手机号码已被注册', 'status': 0}
2019-03-14 20:38:55,875-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'code': '10001', 'msg': '注册成功', 'status': 1} != {'data': None, 'code': '20110', 'msg': '手机号码已被注册', 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:38:55,884-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:55,895-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:55,963-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'pwd': '12345', 'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0}
2019-03-14 20:38:55,970-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:55,983-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,051-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'pwd': '1234567891234567891', 'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0}
2019-03-14 20:38:56,058-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,065-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,131-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'pwd': '123456', 'mobilephone': 'abcd'}，结果是{'data': None, 'code': '20109', 'msg': '手机号码格式不正确', 'status': 0}
2019-03-14 20:38:56,138-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,145-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,225-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'pwd': '123456', 'mobilephone': '13412345679'}，结果是{'data': None, 'code': '20110', 'msg': '手机号码已被注册', 'status': 0}
2019-03-14 20:38:56,234-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,243-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,319-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'data': None, 'code': '10001', 'msg': '登录成功', 'status': 1}
2019-03-14 20:38:56,334-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,343-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,429-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'data': None, 'code': '20111', 'msg': '用户名或密码错误', 'status': 0}
2019-03-14 20:38:56,439-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,449-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,525-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'data': None, 'code': '20111', 'msg': '用户名或密码错误', 'status': 0}
2019-03-14 20:38:56,535-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,546-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,615-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'code': '20103', 'msg': '手机号不能为空', 'status': 0}
2019-03-14 20:38:56,622-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:38:56,629-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:38:56,699-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20103', 'msg': '密码不能为空', 'status': 0}
2019-03-14 20:38:56,706-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:53,542-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:53,617-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'code': '20110', 'status': 0, 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:40:53,636-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'status': 1, 'data': None, 'msg': '注册成功'} != {'code': '20110', 'status': 0, 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:40:53,701-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:53,715-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:53,779-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'status': 0, 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:40:53,830-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:53,838-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:53,909-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'status': 0, 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:40:53,957-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:53,965-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,036-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'code': '20109', 'status': 0, 'data': None, 'msg': '手机号码格式不正确'}
2019-03-14 20:40:54,104-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:54,113-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,188-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'code': '20110', 'status': 0, 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:40:54,242-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:54,251-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,329-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'status': 1, 'data': None, 'msg': '登录成功'}
2019-03-14 20:40:54,378-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:54,386-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,465-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'status': 0, 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:40:54,514-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:54,523-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,594-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'status': 0, 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:40:54,651-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:54,660-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,728-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'status': 0, 'data': None, 'msg': '手机号不能为空'}
2019-03-14 20:40:54,781-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:40:54,790-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:40:54,868-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'status': 0, 'data': None, 'msg': '密码不能为空'}
2019-03-14 20:40:54,919-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:18,296-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:18,370-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'pwd': '123456', 'regname': '自闭患者'}，结果是{'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:41:18,377-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'data': None, 'status': 1, 'msg': '注册成功'} != {'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:41:18,442-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:18,454-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:18,550-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '12345', 'regname': '哎呀'}，结果是{'code': '20108', 'data': None, 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:41:18,598-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:18,607-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:18,703-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '1234567891234567891', 'regname': '妈呀'}，结果是{'code': '20108', 'data': None, 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:41:18,771-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:18,781-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:18,845-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'pwd': '123456', 'regname': '小皮球'}，结果是{'code': '20109', 'data': None, 'status': 0, 'msg': '手机号码格式不正确'}
2019-03-14 20:41:18,894-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:18,902-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:18,969-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'pwd': '123456', 'regname': '小晚晴'}，结果是{'code': '20110', 'data': None, 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:41:19,024-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:19,032-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:19,101-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'code': '10001', 'data': None, 'status': 1, 'msg': '登录成功'}
2019-03-14 20:41:19,162-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:19,170-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:19,244-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'code': '20111', 'data': None, 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:41:19,300-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:19,309-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:19,378-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'code': '20111', 'data': None, 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:41:19,429-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:19,438-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:19,508-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'data': None, 'status': 0, 'msg': '手机号不能为空'}
2019-03-14 20:41:19,568-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:41:19,576-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:41:19,649-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'data': None, 'status': 0, 'msg': '密码不能为空'}
2019-03-14 20:41:19,707-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:06,349-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:06,421-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:42:06,437-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'data': None, 'code': '10001', 'msg': '注册成功'} != {'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:42:06,504-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:06,520-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:06,586-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:42:06,648-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:06,656-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:06,718-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:42:06,769-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:06,777-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:06,870-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'status': 0, 'data': None, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 20:42:06,933-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:06,943-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:07,039-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:42:07,093-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:07,103-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:07,178-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'data': None, 'code': '10001', 'msg': '登录成功'}
2019-03-14 20:42:07,233-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:07,242-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:07,311-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:42:07,368-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:07,377-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:07,448-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:42:07,501-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:07,513-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:07,579-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 20:42:07,634-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:42:07,643-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:42:07,723-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 20:42:07,776-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:48,413-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:48,485-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:43:48,493-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'data': None, 'code': '10001', 'msg': '注册成功'} != {'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:43:48,515-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:48,533-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:48,599-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:43:48,624-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:48,634-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:48,717-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 20:43:48,749-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:48,759-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:48,828-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'status': 0, 'data': None, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 20:43:48,851-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:48,862-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:48,943-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 20:43:48,969-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:48,981-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:49,056-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'data': None, 'code': '10001', 'msg': '登录成功'}
2019-03-14 20:43:49,074-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:49,087-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:49,159-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:43:49,183-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:49,194-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:49,274-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 20:43:49,331-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:49,341-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:49,443-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 20:43:49,466-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:43:49,475-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:43:49,543-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 20:43:49,567-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:05,915-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:05,993-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'status': 0, 'code': '20110', 'msg': '手机号码已被注册', 'data': None}
2019-03-14 20:44:06,007-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'code': '10001', 'msg': '注册成功', 'data': None} != {'status': 0, 'code': '20110', 'msg': '手机号码已被注册', 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:44:06,076-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,090-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:06,185-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18', 'data': None}
2019-03-14 20:44:06,233-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,242-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:06,310-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18', 'data': None}
2019-03-14 20:44:06,362-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,371-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:06,438-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'status': 0, 'code': '20109', 'msg': '手机号码格式不正确', 'data': None}
2019-03-14 20:44:06,494-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,502-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:06,574-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'status': 0, 'code': '20110', 'msg': '手机号码已被注册', 'data': None}
2019-03-14 20:44:06,632-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,641-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:06,745-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'code': '10001', 'msg': '登录成功', 'data': None}
2019-03-14 20:44:06,794-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,803-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:06,881-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'code': '20111', 'msg': '用户名或密码错误', 'data': None}
2019-03-14 20:44:06,934-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:06,942-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:07,014-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'code': '20111', 'msg': '用户名或密码错误', 'data': None}
2019-03-14 20:44:07,066-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:07,074-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:07,142-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'code': '20103', 'msg': '手机号不能为空', 'data': None}
2019-03-14 20:44:07,194-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:44:07,202-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:44:07,282-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'code': '20103', 'msg': '密码不能为空', 'data': None}
2019-03-14 20:44:07,334-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:44,147-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:44,229-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'pwd': '123456', 'mobilephone': '13127750530'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'data': None, 'status': 0}
2019-03-14 20:45:44,236-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'data': None, 'status': 1} != {'code': '20110', 'msg': '手机号码已被注册', 'data': None, 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:45:44,317-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:44,335-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:44,430-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'pwd': '12345', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'data': None, 'status': 0}
2019-03-14 20:45:44,494-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:44,502-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:44,575-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'pwd': '1234567891234567891', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'data': None, 'status': 0}
2019-03-14 20:45:44,624-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:44,632-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:44,701-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'pwd': '123456', 'mobilephone': 'abcd'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'data': None, 'status': 0}
2019-03-14 20:45:44,755-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:44,764-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:44,831-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'pwd': '123456', 'mobilephone': '13412345679'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'data': None, 'status': 0}
2019-03-14 20:45:44,883-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:44,891-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:44,993-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'msg': '登录成功', 'data': None, 'status': 1}
2019-03-14 20:45:45,042-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:45,050-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:45,118-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'data': None, 'status': 0}
2019-03-14 20:45:45,170-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:45,180-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:45,249-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'data': None, 'status': 0}
2019-03-14 20:45:45,305-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:45,313-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:45,378-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'data': None, 'status': 0}
2019-03-14 20:45:45,432-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:45:45,441-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:45:45,545-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'data': None, 'status': 0}
2019-03-14 20:45:45,597-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:36,925-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,006-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'data': None, 'code': '20110', 'msg': '手机号码已被注册', 'status': 0}
2019-03-14 20:46:37,014-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'code': '10001', 'msg': '注册成功', 'status': 1} != {'data': None, 'code': '20110', 'msg': '手机号码已被注册', 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:46:37,073-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,084-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,151-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'data': None, 'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0}
2019-03-14 20:46:37,206-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,214-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,280-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'data': None, 'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0}
2019-03-14 20:46:37,331-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,339-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,405-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'data': None, 'code': '20109', 'msg': '手机号码格式不正确', 'status': 0}
2019-03-14 20:46:37,458-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,466-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,539-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'data': None, 'code': '20110', 'msg': '手机号码已被注册', 'status': 0}
2019-03-14 20:46:37,598-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,606-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,740-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'data': None, 'code': '10001', 'msg': '登录成功', 'status': 1}
2019-03-14 20:46:37,788-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,797-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,862-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'data': None, 'code': '20111', 'msg': '用户名或密码错误', 'status': 0}
2019-03-14 20:46:37,912-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:37,921-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:37,990-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'data': None, 'code': '20111', 'msg': '用户名或密码错误', 'status': 0}
2019-03-14 20:46:38,049-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:38,057-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:38,130-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'code': '20103', 'msg': '手机号不能为空', 'status': 0}
2019-03-14 20:46:38,188-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:46:38,196-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:46:38,262-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20103', 'msg': '密码不能为空', 'status': 0}
2019-03-14 20:46:38,315-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:03,070-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:03,151-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'msg': '手机号码已被注册', 'status': 0, 'code': '20110', 'data': None}
2019-03-14 20:48:03,171-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'status': 1, 'code': '10001', 'data': None} != {'msg': '手机号码已被注册', 'status': 0, 'code': '20110', 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:48:03,236-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:03,250-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:03,336-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'msg': '密码长度必须为6~18', 'status': 0, 'code': '20108', 'data': None}
2019-03-14 20:48:03,394-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:03,402-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:03,478-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'msg': '密码长度必须为6~18', 'status': 0, 'code': '20108', 'data': None}
2019-03-14 20:48:03,525-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:03,534-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:03,613-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'msg': '手机号码格式不正确', 'status': 0, 'code': '20109', 'data': None}
2019-03-14 20:48:03,666-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:03,675-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:03,818-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'msg': '手机号码已被注册', 'status': 0, 'code': '20110', 'data': None}
2019-03-14 20:48:03,865-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:03,873-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:03,993-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'msg': '登录成功', 'status': 1, 'code': '10001', 'data': None}
2019-03-14 20:48:04,050-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:04,059-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:04,144-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'status': 0, 'code': '20111', 'data': None}
2019-03-14 20:48:04,197-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:04,205-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:04,288-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'status': 0, 'code': '20111', 'data': None}
2019-03-14 20:48:04,349-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:04,357-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:04,429-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'status': 0, 'code': '20103', 'data': None}
2019-03-14 20:48:04,479-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:48:04,487-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:48:04,564-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'status': 0, 'code': '20103', 'data': None}
2019-03-14 20:48:04,623-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:56,586-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:56,657-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'data': None, 'code': '20110', 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:49:56,668-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'code': '10001', 'status': 1, 'msg': '注册成功'} != {'data': None, 'code': '20110', 'status': 0, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:49:56,738-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:56,753-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:56,818-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20108', 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:49:56,867-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:56,876-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:56,943-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20108', 'status': 0, 'msg': '密码长度必须为6~18'}
2019-03-14 20:49:57,002-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,011-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,077-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'data': None, 'code': '20109', 'status': 0, 'msg': '手机号码格式不正确'}
2019-03-14 20:49:57,128-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,137-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,205-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'data': None, 'code': '20110', 'status': 0, 'msg': '手机号码已被注册'}
2019-03-14 20:49:57,258-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,267-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,337-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'data': None, 'code': '10001', 'status': 1, 'msg': '登录成功'}
2019-03-14 20:49:57,388-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,397-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,465-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'data': None, 'code': '20111', 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:49:57,522-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,531-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,639-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'data': None, 'code': '20111', 'status': 0, 'msg': '用户名或密码错误'}
2019-03-14 20:49:57,686-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,696-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,762-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'code': '20103', 'status': 0, 'msg': '手机号不能为空'}
2019-03-14 20:49:57,811-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:49:57,820-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:49:57,886-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'code': '20103', 'status': 0, 'msg': '密码不能为空'}
2019-03-14 20:49:57,952-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,102-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,173-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:50:16,180-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:50:16,189-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,198-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,263-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:50:16,271-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,282-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,350-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:50:16,356-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,364-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,455-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'status': 0, 'code': '20109', 'data': None, 'msg': '手机号码格式不正确'}
2019-03-14 20:50:16,462-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,470-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,540-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:50:16,546-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,553-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,623-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'code': '10001', 'data': None, 'msg': '登录成功'}
2019-03-14 20:50:16,629-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,636-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,706-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:50:16,713-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,720-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,817-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:50:16,824-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,836-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,901-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '手机号不能为空'}
2019-03-14 20:50:16,907-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:50:16,914-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:50:16,979-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '密码不能为空'}
2019-03-14 20:50:16,985-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:15,922-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:15,993-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'pwd': '123456', 'regname': '自闭患者'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:52:16,000-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:52:16,008-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,019-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,092-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '12345', 'regname': '哎呀'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:52:16,098-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,107-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,169-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '1234567891234567891', 'regname': '妈呀'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:52:16,176-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,183-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,245-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'pwd': '123456', 'regname': '小皮球'}，结果是{'status': 0, 'code': '20109', 'data': None, 'msg': '手机号码格式不正确'}
2019-03-14 20:52:16,254-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,263-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,333-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'pwd': '123456', 'regname': '小晚晴'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:52:16,340-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,347-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,419-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'status': 1, 'code': '10001', 'data': None, 'msg': '登录成功'}
2019-03-14 20:52:16,425-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,431-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,501-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:52:16,507-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,514-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,581-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:52:16,589-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,596-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,662-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '手机号不能为空'}
2019-03-14 20:52:16,670-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:16,678-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:16,740-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '密码不能为空'}
2019-03-14 20:52:16,746-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:23,500-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:23,575-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 20:52:23,584-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'status': 1, 'data': None} != {'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:52:23,666-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:23,684-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:23,751-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 20:52:23,802-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:23,814-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:23,884-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 20:52:23,960-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:23,968-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,035-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'status': 0, 'data': None}
2019-03-14 20:52:24,104-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:24,113-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,181-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 20:52:24,230-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:24,238-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,309-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'msg': '登录成功', 'status': 1, 'data': None}
2019-03-14 20:52:24,367-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:24,378-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,451-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 20:52:24,501-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:24,510-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,585-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 20:52:24,637-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:24,646-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,715-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'status': 0, 'data': None}
2019-03-14 20:52:24,767-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:52:24,775-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:52:24,845-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'status': 0, 'data': None}
2019-03-14 20:52:24,898-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:54,723-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:54,803-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 20:54:54,810-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'status': 1, 'data': None} != {'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:54:54,823-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:54,838-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:54,908-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 20:54:54,918-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:54,929-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,002-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 20:54:55,011-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,022-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,088-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'status': 0, 'data': None}
2019-03-14 20:54:55,095-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,103-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,173-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 20:54:55,180-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,188-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,287-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'code': '10001', 'msg': '登录成功', 'status': 1, 'data': None}
2019-03-14 20:54:55,294-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,302-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,373-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 20:54:55,381-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,389-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,458-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 20:54:55,467-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,477-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,546-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'status': 0, 'data': None}
2019-03-14 20:54:55,555-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:54:55,565-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:54:55,635-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'status': 0, 'data': None}
2019-03-14 20:54:55,642-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,212-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,289-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:55:14,309-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 20:55:14,356-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,365-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,433-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:55:14,471-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,479-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,541-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 20:55:14,576-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,583-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,647-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'status': 0, 'code': '20109', 'data': None, 'msg': '手机号码格式不正确'}
2019-03-14 20:55:14,681-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,689-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,760-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 20:55:14,800-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,807-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,878-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'status': 1, 'code': '10001', 'data': None, 'msg': '登录成功'}
2019-03-14 20:55:14,915-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:14,922-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:14,994-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:55:15,035-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:15,042-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:15,114-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 20:55:15,152-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:15,160-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:15,225-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '手机号不能为空'}
2019-03-14 20:55:15,270-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 20:55:15,277-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 20:55:15,342-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '密码不能为空'}
2019-03-14 20:55:15,380-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:18,769-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:18,863-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'msg': '手机号码已被注册', 'data': None, 'status': 0, 'code': '20110'}
2019-03-14 21:02:18,872-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'data': None, 'status': 1, 'code': '10001'} != {'msg': '手机号码已被注册', 'data': None, 'status': 0, 'code': '20110'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:02:18,931-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:18,946-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:19,037-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'status': 0, 'code': '20108'}
2019-03-14 21:02:19,085-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:19,095-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:19,199-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'status': 0, 'code': '20108'}
2019-03-14 21:02:19,257-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:19,267-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:19,387-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'msg': '手机号码格式不正确', 'data': None, 'status': 0, 'code': '20109'}
2019-03-14 21:02:19,439-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:19,449-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:19,766-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'msg': '手机号码已被注册', 'data': None, 'status': 0, 'code': '20110'}
2019-03-14 21:02:19,820-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:19,831-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:19,981-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'msg': '登录成功', 'data': None, 'status': 1, 'code': '10001'}
2019-03-14 21:02:20,044-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:20,053-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:20,145-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'msg': '用户名或密码错误', 'data': None, 'status': 0, 'code': '20111'}
2019-03-14 21:02:20,213-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:20,225-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:20,326-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'msg': '用户名或密码错误', 'data': None, 'status': 0, 'code': '20111'}
2019-03-14 21:02:20,393-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:20,405-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:20,532-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'data': None, 'status': 0, 'code': '20103'}
2019-03-14 21:02:20,635-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:02:20,645-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:02:20,756-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'data': None, 'status': 0, 'code': '20103'}
2019-03-14 21:02:20,806-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:33,659-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:33,732-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 21:04:33,751-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:04:33,808-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:33,824-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:33,896-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 21:04:33,948-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:33,956-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,026-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 21:04:34,077-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,085-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,155-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'status': 0, 'code': '20109', 'data': None, 'msg': '手机号码格式不正确'}
2019-03-14 21:04:34,214-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,222-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,322-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 21:04:34,373-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,382-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,451-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'status': 1, 'code': '10001', 'data': None, 'msg': '登录成功'}
2019-03-14 21:04:34,503-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,511-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,582-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 21:04:34,639-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,648-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,722-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 21:04:34,778-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,787-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,856-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '手机号不能为空'}
2019-03-14 21:04:34,913-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:34,921-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:34,990-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '密码不能为空'}
2019-03-14 21:04:35,044-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:51,649-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:51,724-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'data': None, 'status': 0}
2019-03-14 21:04:51,742-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'data': None, 'status': 1} != {'code': '20110', 'msg': '手机号码已被注册', 'data': None, 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:04:51,768-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:51,784-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:51,852-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'data': None, 'status': 0}
2019-03-14 21:04:51,882-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:51,892-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:51,959-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'data': None, 'status': 0}
2019-03-14 21:04:51,979-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:51,989-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,057-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'data': None, 'status': 0}
2019-03-14 21:04:52,077-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:52,087-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,165-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'data': None, 'status': 0}
2019-03-14 21:04:52,189-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:52,198-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,300-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'code': '10001', 'msg': '登录成功', 'data': None, 'status': 1}
2019-03-14 21:04:52,326-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:52,335-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,415-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'data': None, 'status': 0}
2019-03-14 21:04:52,443-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:52,454-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,560-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'data': None, 'status': 0}
2019-03-14 21:04:52,600-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:52,609-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,675-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'data': None, 'status': 0}
2019-03-14 21:04:52,695-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:04:52,704-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:04:52,773-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'data': None, 'status': 0}
2019-03-14 21:04:52,809-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:39,337-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:39,430-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:05:39,452-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'status': 1, 'code': '10001', 'msg': '注册成功'} != {'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:05:39,479-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:39,495-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:39,564-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:05:39,585-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:39,594-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:39,678-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:05:39,698-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:39,707-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:39,798-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 21:05:39,832-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:39,840-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:39,911-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:05:39,933-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:39,942-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:40,022-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'status': 1, 'code': '10001', 'msg': '登录成功'}
2019-03-14 21:05:40,041-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:40,052-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:40,184-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:05:40,207-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:40,216-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:40,301-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:05:40,321-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:40,332-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:40,442-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 21:05:40,464-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:05:40,472-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:05:40,603-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 21:05:40,622-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,329-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:48,409-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
2019-03-14 21:06:48,418-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'data': None, 'code': '10001', 'status': 1} != {'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:06:48,430-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,447-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:48,516-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'code': '20108', 'status': 0}
2019-03-14 21:06:48,540-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,554-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:48,655-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'code': '20108', 'status': 0}
2019-03-14 21:06:48,677-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,685-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:48,757-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'msg': '手机号码格式不正确', 'data': None, 'code': '20109', 'status': 0}
2019-03-14 21:06:48,783-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,793-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:48,866-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
2019-03-14 21:06:48,885-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,894-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:48,964-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'msg': '登录成功', 'data': None, 'code': '10001', 'status': 1}
2019-03-14 21:06:48,987-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:48,995-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:49,072-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'msg': '用户名或密码错误', 'data': None, 'code': '20111', 'status': 0}
2019-03-14 21:06:49,092-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:49,100-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:49,205-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'msg': '用户名或密码错误', 'data': None, 'code': '20111', 'status': 0}
2019-03-14 21:06:49,244-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:49,254-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:49,322-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'data': None, 'code': '20103', 'status': 0}
2019-03-14 21:06:49,345-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:06:49,354-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:06:49,424-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'data': None, 'code': '20103', 'status': 0}
2019-03-14 21:06:49,446-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:32,940-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:33,538-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:07:33,547-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'status': 1, 'code': '10001', 'msg': '注册成功'} != {'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:07:33,574-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:33,586-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:33,837-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:07:33,860-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:33,869-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:35,274-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:07:35,295-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:35,305-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:37,739-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 21:07:37,760-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:37,768-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:41,354-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:07:41,385-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:41,394-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:47,108-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'status': 1, 'code': '10001', 'msg': '登录成功'}
2019-03-14 21:07:47,132-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:47,141-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:48,527-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:07:48,548-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:48,557-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:48,655-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:07:48,685-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:48,695-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:48,907-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 21:07:48,928-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:07:48,937-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:07:49,041-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 21:07:49,066-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:39,900-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,001-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'pwd': '123456', 'regname': '自闭患者'}，结果是{'data': None, 'status': 0, 'msg': '手机号码已被注册', 'code': '20110'}
2019-03-14 21:08:40,011-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'status': 1, 'msg': '注册成功', 'code': '10001'} != {'data': None, 'status': 0, 'msg': '手机号码已被注册', 'code': '20110'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:08:40,040-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,056-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,125-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '12345', 'regname': '哎呀'}，结果是{'data': None, 'status': 0, 'msg': '密码长度必须为6~18', 'code': '20108'}
2019-03-14 21:08:40,146-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,155-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,229-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '1234567891234567891', 'regname': '妈呀'}，结果是{'data': None, 'status': 0, 'msg': '密码长度必须为6~18', 'code': '20108'}
2019-03-14 21:08:40,248-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,258-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,324-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'pwd': '123456', 'regname': '小皮球'}，结果是{'data': None, 'status': 0, 'msg': '手机号码格式不正确', 'code': '20109'}
2019-03-14 21:08:40,343-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,351-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,425-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'pwd': '123456', 'regname': '小晚晴'}，结果是{'data': None, 'status': 0, 'msg': '手机号码已被注册', 'code': '20110'}
2019-03-14 21:08:40,444-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,453-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,523-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'status': 1, 'msg': '登录成功', 'code': '10001'}
2019-03-14 21:08:40,548-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,557-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,626-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'msg': '用户名或密码错误', 'code': '20111'}
2019-03-14 21:08:40,645-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,655-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,725-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'msg': '用户名或密码错误', 'code': '20111'}
2019-03-14 21:08:40,757-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,768-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,845-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'status': 0, 'msg': '手机号不能为空', 'code': '20103'}
2019-03-14 21:08:40,864-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:08:40,874-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:08:40,939-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'msg': '密码不能为空', 'code': '20103'}
2019-03-14 21:08:40,958-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:20,832-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:20,903-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'msg': '手机号码已被注册', 'status': 0, 'code': '20110', 'data': None}
2019-03-14 21:09:20,911-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'status': 1, 'code': '10001', 'data': None} != {'msg': '手机号码已被注册', 'status': 0, 'code': '20110', 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:09:20,923-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:20,933-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:20,998-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'msg': '密码长度必须为6~18', 'status': 0, 'code': '20108', 'data': None}
2019-03-14 21:09:21,007-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,016-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,087-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'msg': '密码长度必须为6~18', 'status': 0, 'code': '20108', 'data': None}
2019-03-14 21:09:21,094-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,100-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,166-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'msg': '手机号码格式不正确', 'status': 0, 'code': '20109', 'data': None}
2019-03-14 21:09:21,177-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,184-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,254-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'msg': '手机号码已被注册', 'status': 0, 'code': '20110', 'data': None}
2019-03-14 21:09:21,261-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,268-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,346-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'msg': '登录成功', 'status': 1, 'code': '10001', 'data': None}
2019-03-14 21:09:21,353-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,360-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,435-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'msg': '用户名或密码错误', 'status': 0, 'code': '20111', 'data': None}
2019-03-14 21:09:21,442-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,449-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,517-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'msg': '用户名或密码错误', 'status': 0, 'code': '20111', 'data': None}
2019-03-14 21:09:21,524-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,531-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,600-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'status': 0, 'code': '20103', 'data': None}
2019-03-14 21:09:21,607-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:21,614-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:21,696-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'status': 0, 'code': '20103', 'data': None}
2019-03-14 21:09:21,703-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,101-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,204-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:09:53,214-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'data': None, 'code': '10001', 'msg': '注册成功'} != {'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:09:53,226-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,235-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,301-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:09:53,309-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,318-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,385-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'status': 0, 'data': None, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:09:53,392-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,400-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,475-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 21:09:53,482-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,491-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,572-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:09:53,580-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,587-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,654-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'status': 1, 'data': None, 'code': '10001', 'msg': '登录成功'}
2019-03-14 21:09:53,661-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,670-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,742-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:09:53,749-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,756-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,825-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:09:53,831-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,838-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:53,931-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 21:09:53,939-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:09:53,946-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:09:54,014-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'data': None, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 21:09:54,021-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,211-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,289-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'msg': '手机号码已被注册', 'code': '20110', 'status': 0, 'data': None}
2019-03-14 21:10:25,296-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'code': '10001', 'status': 1, 'data': None} != {'msg': '手机号码已被注册', 'code': '20110', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:10:25,324-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,339-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,436-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'msg': '密码长度必须为6~18', 'code': '20108', 'status': 0, 'data': None}
2019-03-14 21:10:25,463-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,471-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,539-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'msg': '密码长度必须为6~18', 'code': '20108', 'status': 0, 'data': None}
2019-03-14 21:10:25,559-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,568-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,636-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'msg': '手机号码格式不正确', 'code': '20109', 'status': 0, 'data': None}
2019-03-14 21:10:25,655-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,664-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,742-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'msg': '手机号码已被注册', 'code': '20110', 'status': 0, 'data': None}
2019-03-14 21:10:25,761-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,771-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,840-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'msg': '登录成功', 'code': '10001', 'status': 1, 'data': None}
2019-03-14 21:10:25,860-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,869-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:25,937-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'msg': '用户名或密码错误', 'code': '20111', 'status': 0, 'data': None}
2019-03-14 21:10:25,957-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:25,966-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:26,041-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'msg': '用户名或密码错误', 'code': '20111', 'status': 0, 'data': None}
2019-03-14 21:10:26,060-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:26,069-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:26,138-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'code': '20103', 'status': 0, 'data': None}
2019-03-14 21:10:26,164-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:26,175-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:26,242-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'code': '20103', 'status': 0, 'data': None}
2019-03-14 21:10:26,260-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:45,274-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:45,371-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 21:10:45,394-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'status': 1, 'data': None} != {'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:10:45,442-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:45,453-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:45,579-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 21:10:45,620-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:45,628-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:45,767-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 21:10:45,802-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:45,810-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,009-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'status': 0, 'data': None}
2019-03-14 21:10:46,051-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:46,059-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,188-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 21:10:46,224-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:46,232-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,309-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'code': '10001', 'msg': '登录成功', 'status': 1, 'data': None}
2019-03-14 21:10:46,358-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:46,366-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,442-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 21:10:46,480-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:46,488-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,577-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 21:10:46,615-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:46,623-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,724-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'status': 0, 'data': None}
2019-03-14 21:10:46,761-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:10:46,771-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:10:46,848-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'status': 0, 'data': None}
2019-03-14 21:10:46,888-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:10,601-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:10,672-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'mobilephone': '13127750530', 'pwd': '123456'}，结果是{'data': None, 'msg': '手机号码已被注册', 'code': '20110', 'status': 0}
2019-03-14 21:11:10,679-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'msg': '注册成功', 'code': '10001', 'status': 1} != {'data': None, 'msg': '手机号码已被注册', 'code': '20110', 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:11:10,756-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:10,770-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:10,834-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'mobilephone': '13412345678', 'pwd': '12345'}，结果是{'data': None, 'msg': '密码长度必须为6~18', 'code': '20108', 'status': 0}
2019-03-14 21:11:10,886-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:10,895-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:10,965-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'mobilephone': '13412345678', 'pwd': '1234567891234567891'}，结果是{'data': None, 'msg': '密码长度必须为6~18', 'code': '20108', 'status': 0}
2019-03-14 21:11:11,015-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,025-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,095-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'mobilephone': 'abcd', 'pwd': '123456'}，结果是{'data': None, 'msg': '手机号码格式不正确', 'code': '20109', 'status': 0}
2019-03-14 21:11:11,146-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,155-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,227-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'mobilephone': '13412345679', 'pwd': '123456'}，结果是{'data': None, 'msg': '手机号码已被注册', 'code': '20110', 'status': 0}
2019-03-14 21:11:11,279-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,288-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,375-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'msg': '登录成功', 'code': '10001', 'status': 1}
2019-03-14 21:11:11,431-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,441-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,516-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'msg': '用户名或密码错误', 'code': '20111', 'status': 0}
2019-03-14 21:11:11,569-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,577-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,649-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'msg': '用户名或密码错误', 'code': '20111', 'status': 0}
2019-03-14 21:11:11,699-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,708-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,780-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'msg': '手机号不能为空', 'code': '20103', 'status': 0}
2019-03-14 21:11:11,845-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:11,854-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:11,919-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'msg': '密码不能为空', 'code': '20103', 'status': 0}
2019-03-14 21:11:11,970-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:56,933-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,014-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'pwd': '123456', 'mobilephone': '13127750530'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 21:11:57,022-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'status': 1, 'data': None} != {'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:11:57,087-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,104-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,172-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'pwd': '12345', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 21:11:57,224-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,235-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,301-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'pwd': '1234567891234567891', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 21:11:57,354-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,363-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,429-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'pwd': '123456', 'mobilephone': 'abcd'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'status': 0, 'data': None}
2019-03-14 21:11:57,480-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,489-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,567-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'pwd': '123456', 'mobilephone': '13412345679'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 21:11:57,622-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,632-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,710-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'msg': '登录成功', 'status': 1, 'data': None}
2019-03-14 21:11:57,760-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,769-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,843-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 21:11:57,897-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:57,905-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:57,976-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 21:11:58,039-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:58,048-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:58,117-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'status': 0, 'data': None}
2019-03-14 21:11:58,173-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:11:58,182-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:11:58,247-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'status': 0, 'data': None}
2019-03-14 21:11:58,303-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:12:44,986-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:06,005-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:06,125-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:14,053-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'code': '20108', 'status': 0}
2019-03-14 21:13:14,133-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:14,140-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:15,876-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'msg': '密码长度必须为6~18', 'data': None, 'code': '20108', 'status': 0}
2019-03-14 21:13:15,951-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:15,958-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:17,356-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'msg': '手机号码格式不正确', 'data': None, 'code': '20109', 'status': 0}
2019-03-14 21:13:17,439-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:17,447-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:18,130-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'msg': '手机号码已被注册', 'data': None, 'code': '20110', 'status': 0}
2019-03-14 21:13:18,213-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:18,226-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:18,930-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'msg': '登录成功', 'data': None, 'code': '10001', 'status': 1}
2019-03-14 21:13:19,001-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:19,008-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:20,691-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'data': None, 'code': '20111', 'status': 0}
2019-03-14 21:13:20,770-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:20,778-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:22,534-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'data': None, 'code': '20111', 'status': 0}
2019-03-14 21:13:22,625-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:22,633-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:25,290-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'data': None, 'code': '20103', 'status': 0}
2019-03-14 21:13:25,380-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:13:25,388-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:13:26,476-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'data': None, 'code': '20103', 'status': 0}
2019-03-14 21:13:26,579-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:07,947-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:08,083-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127750530'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:14:08,090-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'status': 1, 'code': '10001', 'msg': '注册成功'} != {'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:14:08,173-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:08,184-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:08,331-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:14:08,411-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:08,418-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:08,631-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:14:08,701-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:08,708-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:08,933-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'data': None, 'status': 0, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 21:14:09,005-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:09,015-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:09,194-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:14:09,267-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:09,275-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:09,350-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'data': None, 'status': 1, 'code': '10001', 'msg': '登录成功'}
2019-03-14 21:14:09,423-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:09,431-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:09,502-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:14:09,573-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:09,580-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:09,694-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:14:09,760-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:09,767-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:09,944-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 21:14:10,019-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:10,027-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:10,366-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 21:14:10,431-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:29,675-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:29,848-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'pwd': '123456', 'regname': '自闭患者'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:14:29,856-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'data': None, 'status': 1, 'code': '10001', 'msg': '注册成功'} != {'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:14:29,880-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:29,899-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:29,992-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '12345', 'regname': '哎呀'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:14:30,017-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,028-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,131-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '1234567891234567891', 'regname': '妈呀'}，结果是{'data': None, 'status': 0, 'code': '20108', 'msg': '密码长度必须为6~18'}
2019-03-14 21:14:30,154-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,163-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,233-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'pwd': '123456', 'regname': '小皮球'}，结果是{'data': None, 'status': 0, 'code': '20109', 'msg': '手机号码格式不正确'}
2019-03-14 21:14:30,256-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,266-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,440-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'pwd': '123456', 'regname': '小晚晴'}，结果是{'data': None, 'status': 0, 'code': '20110', 'msg': '手机号码已被注册'}
2019-03-14 21:14:30,465-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,474-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,550-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'status': 1, 'code': '10001', 'msg': '登录成功'}
2019-03-14 21:14:30,574-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,584-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,661-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:14:30,681-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,694-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,773-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20111', 'msg': '用户名或密码错误'}
2019-03-14 21:14:30,794-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,805-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,876-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '手机号不能为空'}
2019-03-14 21:14:30,895-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:30,906-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:30,973-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'status': 0, 'code': '20103', 'msg': '密码不能为空'}
2019-03-14 21:14:30,992-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:44,370-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:44,444-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'status': 0, 'msg': '手机号码已被注册', 'code': '20110', 'data': None}
2019-03-14 21:14:44,451-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'msg': '注册成功', 'code': '10001', 'data': None} != {'status': 0, 'msg': '手机号码已被注册', 'code': '20110', 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:14:44,486-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:44,500-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:44,572-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'status': 0, 'msg': '密码长度必须为6~18', 'code': '20108', 'data': None}
2019-03-14 21:14:44,595-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:44,604-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:44,674-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'status': 0, 'msg': '密码长度必须为6~18', 'code': '20108', 'data': None}
2019-03-14 21:14:44,694-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:44,703-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:44,771-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'status': 0, 'msg': '手机号码格式不正确', 'code': '20109', 'data': None}
2019-03-14 21:14:44,791-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:44,800-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:44,875-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'status': 0, 'msg': '手机号码已被注册', 'code': '20110', 'data': None}
2019-03-14 21:14:44,894-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:44,903-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:44,977-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'msg': '登录成功', 'code': '10001', 'data': None}
2019-03-14 21:14:44,996-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:45,004-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:45,078-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'msg': '用户名或密码错误', 'code': '20111', 'data': None}
2019-03-14 21:14:45,097-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:45,106-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:45,174-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'msg': '用户名或密码错误', 'code': '20111', 'data': None}
2019-03-14 21:14:45,194-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:45,204-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:45,271-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'msg': '手机号不能为空', 'code': '20103', 'data': None}
2019-03-14 21:14:45,296-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:14:45,306-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:14:45,380-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'msg': '密码不能为空', 'code': '20103', 'data': None}
2019-03-14 21:14:45,400-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,005-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,089-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127750530', 'regname': '自闭患者', 'pwd': '123456'}，结果是{'msg': '手机号码已被注册', 'code': '20110', 'data': None, 'status': 0}
2019-03-14 21:15:05,098-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'code': '10001', 'data': None, 'status': 1} != {'msg': '手机号码已被注册', 'code': '20110', 'data': None, 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:15:05,123-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,147-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,218-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '哎呀', 'pwd': '12345'}，结果是{'msg': '密码长度必须为6~18', 'code': '20108', 'data': None, 'status': 0}
2019-03-14 21:15:05,244-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,255-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,321-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'regname': '妈呀', 'pwd': '1234567891234567891'}，结果是{'msg': '密码长度必须为6~18', 'code': '20108', 'data': None, 'status': 0}
2019-03-14 21:15:05,344-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,353-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,448-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'regname': '小皮球', 'pwd': '123456'}，结果是{'msg': '手机号码格式不正确', 'code': '20109', 'data': None, 'status': 0}
2019-03-14 21:15:05,480-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,489-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,571-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'regname': '小晚晴', 'pwd': '123456'}，结果是{'msg': '手机号码已被注册', 'code': '20110', 'data': None, 'status': 0}
2019-03-14 21:15:05,592-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,603-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,680-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'msg': '登录成功', 'code': '10001', 'data': None, 'status': 1}
2019-03-14 21:15:05,699-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,714-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,786-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'code': '20111', 'data': None, 'status': 0}
2019-03-14 21:15:05,805-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,815-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,888-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'msg': '用户名或密码错误', 'code': '20111', 'data': None, 'status': 0}
2019-03-14 21:15:05,908-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:05,917-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:05,991-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'code': '20103', 'data': None, 'status': 0}
2019-03-14 21:15:06,019-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:06,031-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:06,108-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'code': '20103', 'data': None, 'status': 0}
2019-03-14 21:15:06,127-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:28,098-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:28,177-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127750530', 'regname': '自闭患者'}，结果是{'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
2019-03-14 21:15:28,184-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1} != {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:15:28,278-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:28,299-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:28,364-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'code': '20108', 'data': None, 'msg': '密码长度必须为6~18', 'status': 0}
2019-03-14 21:15:28,434-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:28,442-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:28,508-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'code': '20108', 'data': None, 'msg': '密码长度必须为6~18', 'status': 0}
2019-03-14 21:15:28,583-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:28,590-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:28,654-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'code': '20109', 'data': None, 'msg': '手机号码格式不正确', 'status': 0}
2019-03-14 21:15:28,728-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:28,736-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:28,805-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
2019-03-14 21:15:28,874-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:28,882-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:28,959-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'data': None, 'msg': '登录成功', 'status': 1}
2019-03-14 21:15:29,029-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:29,036-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:29,106-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'data': None, 'msg': '用户名或密码错误', 'status': 0}
2019-03-14 21:15:29,213-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:29,220-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:29,285-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'data': None, 'msg': '用户名或密码错误', 'status': 0}
2019-03-14 21:15:29,354-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:29,362-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:29,429-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'data': None, 'msg': '手机号不能为空', 'status': 0}
2019-03-14 21:15:29,501-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:29,509-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:29,572-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'data': None, 'msg': '密码不能为空', 'status': 0}
2019-03-14 21:15:29,641-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:52,184-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:52,265-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13127758530', 'pwd': '123456', 'regname': '自闭患者'}，结果是{'data': None, 'msg': '注册成功', 'status': 1, 'code': '10001'}
2019-03-14 21:15:52,349-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:52,356-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:52,418-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '12345', 'regname': '哎呀'}，结果是{'data': None, 'msg': '密码长度必须为6~18', 'status': 0, 'code': '20108'}
2019-03-14 21:15:52,490-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:52,500-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:52,567-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678', 'pwd': '1234567891234567891', 'regname': '妈呀'}，结果是{'data': None, 'msg': '密码长度必须为6~18', 'status': 0, 'code': '20108'}
2019-03-14 21:15:52,656-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:52,664-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:52,731-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': 'abcd', 'pwd': '123456', 'regname': '小皮球'}，结果是{'data': None, 'msg': '手机号码格式不正确', 'status': 0, 'code': '20109'}
2019-03-14 21:15:52,805-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:52,814-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:52,885-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345679', 'pwd': '123456', 'regname': '小晚晴'}，结果是{'data': None, 'msg': '手机号码已被注册', 'status': 0, 'code': '20110'}
2019-03-14 21:15:52,953-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:52,961-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:53,028-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13512365669', 'pwd': '123456'}，结果是{'data': None, 'msg': '登录成功', 'status': 1, 'code': '10001'}
2019-03-14 21:15:53,100-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:53,108-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:53,176-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '1341234567', 'pwd': '123456'}，结果是{'data': None, 'msg': '用户名或密码错误', 'status': 0, 'code': '20111'}
2019-03-14 21:15:53,250-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:53,258-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:53,329-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '134123456789', 'pwd': '123456'}，结果是{'data': None, 'msg': '用户名或密码错误', 'status': 0, 'code': '20111'}
2019-03-14 21:15:53,408-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:53,416-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:53,488-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'data': None, 'msg': '手机号不能为空', 'status': 0, 'code': '20103'}
2019-03-14 21:15:53,568-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:15:53,576-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:15:53,641-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'data': None, 'msg': '密码不能为空', 'status': 0, 'code': '20103'}
2019-03-14 21:15:53,712-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:10,571-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:10,959-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13127758530', 'regname': '自闭患者'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 21:16:10,965-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:16:11,047-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:11,059-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:11,286-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'mobilephone': '13412345678', 'regname': '哎呀'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 21:16:11,359-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:11,366-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:11,576-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'mobilephone': '13412345678', 'regname': '妈呀'}，结果是{'status': 0, 'code': '20108', 'data': None, 'msg': '密码长度必须为6~18'}
2019-03-14 21:16:11,660-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:11,667-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:11,991-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': 'abcd', 'regname': '小皮球'}，结果是{'status': 0, 'code': '20109', 'data': None, 'msg': '手机号码格式不正确'}
2019-03-14 21:16:12,064-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:12,072-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:12,633-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13412345679', 'regname': '小晚晴'}，结果是{'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
2019-03-14 21:16:12,713-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:12,720-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:13,244-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'status': 1, 'code': '10001', 'data': None, 'msg': '登录成功'}
2019-03-14 21:16:13,320-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:13,328-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:13,544-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 21:16:13,619-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:13,627-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:13,981-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2019-03-14 21:16:14,056-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:14,064-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:14,360-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '手机号不能为空'}
2019-03-14 21:16:14,434-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:16:14,443-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:16:14,691-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'status': 0, 'code': '20103', 'data': None, 'msg': '密码不能为空'}
2019-03-14 21:16:14,759-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:47,620-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:47,688-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '自闭患者', 'mobilephone': '13127758530'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 21:24:47,700-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'code': '10001', 'msg': '注册成功', 'status': 1, 'data': None} != {'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:24:47,776-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:47,782-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:47,858-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '12345', 'regname': '哎呀', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 21:24:47,925-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:47,931-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:47,999-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '1234567891234567891', 'regname': '妈呀', 'mobilephone': '13412345678'}，结果是{'code': '20108', 'msg': '密码长度必须为6~18', 'status': 0, 'data': None}
2019-03-14 21:24:48,077-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:48,083-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:48,154-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小皮球', 'mobilephone': 'abcd'}，结果是{'code': '20109', 'msg': '手机号码格式不正确', 'status': 0, 'data': None}
2019-03-14 21:24:48,240-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:48,247-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:48,320-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'regname': '小晚晴', 'mobilephone': '13412345679'}，结果是{'code': '20110', 'msg': '手机号码已被注册', 'status': 0, 'data': None}
2019-03-14 21:24:48,390-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:48,396-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:48,464-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'code': '10001', 'msg': '登录成功', 'status': 1, 'data': None}
2019-03-14 21:24:48,545-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:48,551-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:48,631-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 21:24:48,701-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:48,707-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:48,772-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'code': '20111', 'msg': '用户名或密码错误', 'status': 0, 'data': None}
2019-03-14 21:24:48,853-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:48,859-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:48,921-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'code': '20103', 'msg': '手机号不能为空', 'status': 0, 'data': None}
2019-03-14 21:24:48,994-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:24:49,000-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:24:49,060-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'code': '20103', 'msg': '密码不能为空', 'status': 0, 'data': None}
2019-03-14 21:24:49,142-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:07,938-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:08,834-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '自闭患者', 'pwd': '123456', 'mobilephone': '13127758530'}，结果是{'msg': '手机号码已被注册', 'code': '20110', 'status': 0, 'data': None}
2019-03-14 21:25:08,844-DEBUG-loger_class.py-py14-日志信息:断言出错啦，错误是{'msg': '注册成功', 'code': '10001', 'status': 1, 'data': None} != {'msg': '手机号码已被注册', 'code': '20110', 'status': 0, 'data': None}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

2019-03-14 21:25:08,921-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:08,927-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:09,039-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '哎呀', 'pwd': '12345', 'mobilephone': '13412345678'}，结果是{'msg': '密码长度必须为6~18', 'code': '20108', 'status': 0, 'data': None}
2019-03-14 21:25:09,113-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:09,118-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:10,193-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '妈呀', 'pwd': '1234567891234567891', 'mobilephone': '13412345678'}，结果是{'msg': '密码长度必须为6~18', 'code': '20108', 'status': 0, 'data': None}
2019-03-14 21:25:10,274-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:10,280-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:12,281-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小皮球', 'pwd': '123456', 'mobilephone': 'abcd'}，结果是{'msg': '手机号码格式不正确', 'code': '20109', 'status': 0, 'data': None}
2019-03-14 21:25:12,367-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:12,373-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:12,464-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'regname': '小晚晴', 'pwd': '123456', 'mobilephone': '13412345679'}，结果是{'msg': '手机号码已被注册', 'code': '20110', 'status': 0, 'data': None}
2019-03-14 21:25:12,532-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:12,539-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:12,688-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '13512365669'}，结果是{'msg': '登录成功', 'code': '10001', 'status': 1, 'data': None}
2019-03-14 21:25:12,763-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:12,770-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:12,899-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '1341234567'}，结果是{'msg': '用户名或密码错误', 'code': '20111', 'status': 0, 'data': None}
2019-03-14 21:25:12,966-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:12,972-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:13,102-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456', 'mobilephone': '134123456789'}，结果是{'msg': '用户名或密码错误', 'code': '20111', 'status': 0, 'data': None}
2019-03-14 21:25:13,168-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:13,175-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:13,294-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'pwd': '123456'}，结果是{'msg': '手机号不能为空', 'code': '20103', 'status': 0, 'data': None}
2019-03-14 21:25:13,360-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
2019-03-14 21:25:13,368-DEBUG-loger_class.py-py14-日志信息:.....开始执行条测试用例啦.....
2019-03-14 21:25:13,453-DEBUG-loger_class.py-py14-日志信息:正在执行的数据是{'mobilephone': '13412345678'}，结果是{'msg': '密码不能为空', 'code': '20103', 'status': 0, 'data': None}
2019-03-14 21:25:13,524-DEBUG-loger_class.py-py14-日志信息:.....用例执行完毕啦.....
